src/python/pull_gsheets_translations_mvp.py

```diff
@@ -0,0 +1,299 @@
+# PACKAGES
+#####################
+#https://docs.python.org/3/tutorial/modules.html
+from json import dump
+from gspread import authorize
+from google.oauth2.service_account import Credentials
+from pandas import Series, DataFrame
+from os import path, mkdir
+from string import punctuation
+
+from nltk import pos_tag, download
+from nltk.tokenize import word_tokenize
+download('punkt')
+download('stopwords')
+download('averaged_perceptron_tagger')
+
+#####################
+# CONSTANTS
+#####################
+scope = ['https://spreadsheets.google.com/feeds',
+'https://www.googleapis.com/auth/drive']
+
+CREDENTIALS_PATH = '../../.my.goauth'
+TRANSLATION_SHEETS_REGEX = ' - Master Sheet'
+TRANSLATION_SHEETS_REGEX_OLD = "(OLD)"
+OUT_DIR = '../../client/public/locales/'
+#OUT_DIR = 'translations/'
+LANGUAGES_TO_PULL = ['English','Dutch (Netherlands)','Spanish','Italian','French','Russian']
+LANGUAGE_LETTERS_DICT = {
+	'English':'en',
+	'Dutch':'nl',
+	'Spanish':'es',
+	'Italian':'it',
+	'French':'fr',
+	'Russian':'ru'
+}
+
+def convert_to_camelCase(value):
+	""" Converts a string to camelCase and removes punctuation.
+
+	Arguments:
+		value {[string]} -- [string from the 'value' column in language_df]
+
+	Returns:
+		[string] -- [returns a string with no punctuation, in camelCase format.]
+	"""
+	# if value is a string with more than one word
+	if len(value.split()) > 1:
+		#converts value to camelCase
+		camelCase = value.split()[0].lower() + " ".join(value.split()[1:]).title().replace(" ","")
+		#removes punctuation
+		return camelCase.translate(str.maketrans('', '', punctuation))
+	else:
+		return value.lower().translate(str.maketrans('', '', punctuation))
+
+def depunctuate(text):
+	""" Removes punctuation from text.
+
+	Arguments:
+		text {[string]} -- [any string]
+
+	Returns:
+		[string] -- [string with punctuation removed]
+	"""
+	chars = punctuation
+	for c in chars:
+		text = text.replace(c, "")
+	return text
+
+def education_value_cleaner(language_df):
+	""" Converts the long text in the Education 'value' column to shorter text to be used as JSON keys.
+
+	Arguments:
+		language_df {[Series]} -- [A pandas series containing the 'value' column from the 'Education' sheet ]
+
+	Returns:
+		[Series] -- [A pandas Series with shortened text and no punctuation.]
+	"""
+	lst = [word_tokenize(x) for x in language_df.value.tolist()]
+	#http://www.nltk.org/book_1ed/ch05.html
+	pos_tagged = [pos_tag(x) for x in lst]
+	lst_new=[]
+	for pt in pos_tagged:
+		#choose nouns in sentence
+		nouns = [depunctuate(x[0]) for x in pt if "NN" in x[1]]
+		#choose verbs in setence
+		verbs = [depunctuate(x[0]) for x in pt if 'VB' in x[1]]
+		#make fieldKey and add to new list of fieldKeys
+		if len(verbs)==0 and len(nouns)==2:
+			lst_new.append(nouns[0].lower() + nouns[1].title())
+		elif len(verbs)==1 and len(nouns)==1:
+			lst_new.append(nouns[0].lower() + verbs[0].title())
+		else:
+			lst_new.append(nouns[0].lower() + nouns[1].title() + verbs[0].title())

```

Comment:
This is a bit fragile, but if it works with the current sheets it should be fine. We can fix it later